common/src/main/java/io/github/steveplays28/blinkload/client/cache/BlinkLoadCache.java

```diff
@@ -105,6 +105,9 @@ public static void cacheData(@NotNull StitchResult stitchResult) {
 		try (@NotNull Reader reader = new FileReader(CACHED_DATA_FILE)) {
 			// Convert the JSON data to a Java object
 			@NotNull var stitchResults = JsonUtil.getGson().fromJson(reader, StitchResult[].class);
+			if (stitchResults == null) {
+				return new ConcurrentHashMap<>();
```

Comment:
I'm not sure whether should `return cachedData` here. I also used Copilot on https://github.com/steves-underwater-paradise/blinkload/issues/11, and got the same fix as the issue above, but with a different return statement. Maybe we can solve all 4 issues at the same time.

https://github.com/steves-underwater-paradise/blinkload/issues/11